corehq/ex-submodules/casexml/apps/case/models.py

```diff
@@ -219,6 +224,10 @@ def is_deleted(self):
     def has_indices(self):
         return self.indices or self.reverse_indices
 
+    @property
+    def closed_transactions(self):
```

Comment:
@snopoke this is defined on the CommCaseCase model. i just want to double check that it's fine to define properties on it without having clashing errors